### PR TITLE
chore(FDS-399):[Node]: Update node engine version to 14.18.2

### PR DESCRIPTION
**Pull request by rociobaena**
Update node engine version to 14.18.2

### Dependencies
Update node version to the most recent version.

image
image

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

File: utils/eslint-config/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "eslint-plugin-testing-library": "^4.12.4"
   },
   "engines": {
-    "node": "^10.12.0 || >=12.0.0"
+    "node": "^10.12.0 || >=14.18.2"
   },
```
Comment (brianespinosa) [inline utils/eslint-config/package.json:43]: We can go ahead and remove this engines completely. We should not be building this package from within the package directory. And we should definitely not still allow node 10.

**Review by brianespinosa — CHANGES_REQUESTED** [review]
One small change needed to the eslint package. Other than that, we just need to verify that when Vercel is building, we can see in the log output that they are now using node 14 as well. They _should_ be based on the package.json engines defined in the /docs package, but we want to check, just in case.

**Comment by rociobaena** [comment]
> One small change needed to the eslint package. Other than that, we just need to verify that when Vercel is building, we can see in the log output that they are now using node 14 as well. They _should_ be based on the package.json engines defined in the /docs package, but we want to check, just in case.

Hi Brian.
This is from the Vercel settings:
image

File: utils/eslint-config/package.json (hunk 1 above)
Comment (rociobaena) [reply utils/eslint-config/package.json]: Engine removed from package.

**Review by brianespinosa — APPROVED** [review]
Thanks for doing this, @rociobaena!

_SO_ much easier than the Node 10 -> 12 update. :)

**Review by lordpixel — APPROVED** [review]
All good @rociobaena, I'd only ask to use conventional commits format for all commits. 👍

**Pull request merged**